test/functional/test_runner.py

```diff
@@ -489,7 +489,7 @@ def check_script_list(src_dir):
     Check that there are no scripts in the functional tests directory which are
     not being run by pull-tester.py."""
     script_dir = src_dir + '/test/functional/'
-    python_files = set([t for t in os.listdir(script_dir) if t[-3:] == ".py"])
+    python_files = set([file for file in os.listdir(script_dir) if file[-3:] == ".py"])
```

Comment:
file.endswith(".py") is slightly more readable

Reply:
@mcdallas Agreed and updated. Thanks!